charts/images.yaml

```diff
@@ -204,7 +204,7 @@ images:
 - name: fluent-bit-plugin-installer
   sourceRepository: github.com/gardener/logging
   repository: eu.gcr.io/gardener-project/gardener/fluent-bit-to-loki
-  tag: "v0.28.0"
+  tag: "v0.30.0"
```

Comment:
Bugged version of fluent-bit-to-loki plugin.

Reply:
/hold